botocore/docs/utils.py

```diff
@@ -177,3 +178,18 @@ def append_documentation(self, event_name, section, **kwargs):
             description_section = section.get_section(
                 'param-documentation')
             description_section.writeln(self._doc_string)
+
+
+_CONTROLS = {
+    '\n': '\\n',
```

Comment:
I *think* it's just characters that mess with whitespace. I'll try the examples you gave.